favoritespanewidget.cc

```diff
@@ -906,7 +907,7 @@ void FavoritesModel::removeItemsForIndexes( const QModelIndexList & idxList )
   for( int i = lowestLevel; i >= 0; i-- )
   {
     QModelIndexList idxSublist = itemsToDelete[ i ];
-    qSort( idxSublist.begin(), idxSublist.end(), qGreater< QModelIndex >() );
+    std::sort( idxSublist.begin(), idxSublist.end(), qGreater< QModelIndex >() );
```

Comment:
`qGreater` is deprecated too. Might as well fix it too along with another usage below.